togeojson.js

```diff
@@ -234,6 +235,10 @@ var toGeoJSON = (function() {
                     var end = nodeVal(get1(timeSpan, 'end'));
                     properties.timespan = { begin: begin, end: end };
                 }
+                if (timeStamp) {
+                    var when = nodeVal(get1(timeStamp, 'when'));
+                    properties.timestamp = { when: when };
```

Comment:
could these two lines instead be

```js
properties.timestamp = nodeVal(get1(timeStamp, 'when'));
```

In JavaScript, there's usually not a strong reason to wrap a value in an object like this.